src/components/entries/FEEL/FeelIndicator.js

```diff
@@ -0,0 +1,13 @@
+export function FeelIndicator(props) {
+  const {
+    active
+  } = props;
+
+  if (!active) {
+    return null;
+  }
+
+  return <span class="bio-properties-panel-feel-indicator">
+    <button disabled>=</button>
```

Comment:
I'm wondering: why does it have to be a (disabled) button? Will it ever do something or is it just because of the styles?

Reply:
Oh, good catch. In the beginning the Indicator was used to switch and somehow the disabled button is a relict of this. I will replace it, styling should work with any other tag as well 👍